docs/configure

```diff
@@ -867,22 +867,10 @@ virtual_delivery:
   mode = 0600
   maildir_format = true
   create_directory = true
-  directory = ${lookup mysql{select smtp from users,domains \
-		where localpart = '${quote_mysql:$local_part}' \
-		and domain = '${quote_mysql:$domain}' \
-		and users.domain_id = domains.domain_id}}
-  user = ${lookup mysql{select users.uid  from users,domains \
-		where localpart = '${quote_mysql:$local_part}' \
-		and domain = '${quote_mysql:$domain}' \
-		and users.domain_id = domains.domain_id}}
-  group = ${lookup mysql{select users.gid from users,domains \
-		where localpart = '${quote_mysql:$local_part}' \
-		and domain = '${quote_mysql:$domain}' \
-		and users.domain_id = domains.domain_id}}
-  quota = ${lookup mysql{select users.quota from users,domains \
-  		where localpart = '${quote_mysql:$local_part}' \
-		and domain = '${quote_mysql:$domain}' \
-		and users.domain_id = domains.domain_id}{${value}M}}
+  directory = ${extract{smtp}{$address_data}}
```

Comment:
No, we don't need it: https://books.google.fr/books?id=foCRVaMeRMgC&pg=PA106&lpg=PA106&dq=exim+address_data&source=bl&ots=wrrhBdPqOU&sig=grP98HExo1dLJjQHxROB9dnNK-0&hl=de&sa=X&ved=0CDQQ6AEwAmoVChMIs4zl7e_vyAIVQewUCh364gkZ#v=onepage&q=exim%20address_data&f=false

Like this, we reduce the number of lookups and get a compact, readable code. I don't see the point why we should run the same query twice.